app/views/entities/show.rb

```diff
@@ -38,24 +39,28 @@ def include_nav_searchbar
       true
     end
 
-    # TODO
-    # def head_meta
-    #   mustache[:head_meta] ||= begin
-    #     title = page_title
-    #     head_meta = [
-    #       { meta_name: 'description', content: head_meta_description },
-    #       { meta_property: 'fb:appid', content: '185778248173748' },
-    #       { meta_name: 'twitter:card', content: 'summary' },
-    #       { meta_name: 'twitter:site', content: '@EuropeanaEU' },
-    #       { meta_property: 'og:sitename', content: title },
-    #       { meta_property: 'og:description', content: head_meta_description },
-    #       { meta_property: 'og:url', content: collection_url(@collection.key) }
-    #     ]
-    #     head_meta << { meta_property: 'og:title', content: title } unless title.nil?
-    #     head_meta << { meta_property: 'og:image', content: @landing_page.og_image } unless @landing_page.og_image.blank?
-    #     head_meta + super
-    #   end
-    # end
+    def head_meta
+      mustache[:head_meta] ||= begin
+        title = page_title
+        description = truncate(entity_description, length: 350, separator: ' ')
+        description = description.strip! || description
```

Comment:
rather than do this here at the top of the method, can we implement a new method for head_meta_description that also assigns the head_meta_description into the mustache data structure same as in views/collections/show.rb ?